burn-tensor/src/tests/ops/mul.rs

```diff
@@ -30,6 +30,17 @@ mod tests {
         assert_eq!(data_expected, data_actual);
     }
 
+    #[test]
+    fn test_mul_broadcast_2_dims() {
```

Comment:
do we care about adding a test for `Int`?

Reply:
I don't think it's necessary, even if tch supports it, in Burn we do not expose int matmul in our API

Reply:
This is a normal mul that is available to the int tensor, but theorically this test validates the broadcasting logic that is present for all binary operations.